mlir/test/Conversion/TosaToLinalg/tosa-to-linalg-invalidperm.mlir

```diff
@@ -0,0 +1,10 @@
+// RUN: not --crash mlir-opt %s --pass-pipeline="builtin.module(func.func(tosa-to-linalg-named))" 2>&1 | FileCheck -check-prefix=CHECK %s
+
+func.func @func1() {
+	%arg0 = tensor.empty() : tensor<3x4x5xi32>
```

Comment:
You've used very casual names for your functions and variables. This is not recommended.

Reply:
I will modify.